### DISCUSSION TITLE
ValueError: Unexpected value for 'parser'. Options are: 'spacy', 'stanza', 'udpipe'

### DISCUSSION THREAD

**Discussion by sudha-kannan** [explosion/spaCy#10511] [Help: Coding & Implementations] [answered]
As per the init_parser definition the example provided in the spacy_conll page has mixed up the :  "model_or_lang"
 and  "parser" parameters.

**Comment by ljvmiranda921** [comment, marked as answer]
Hi @sudha-kannan , thanks for reporting. The documentation is indeed incorrect, I'll put up a fix in a few.
For posterity, `model_or_lang` and `parser` are keyword arguments, so you can avoid this by explicitly assigning values to each.

**Edit**: Should be fixed now: https://github.com/explosion/spaCy/pull/10516
